packages/taquito-michel-codec/src/utils.ts

```diff
@@ -173,7 +173,7 @@ export interface UnpackAnnotationsOptions {
     emptyFields?: boolean;
 }
 
-const annRe = /^(@%|@%%|%@|[@:%]([_0-9a-zA-Z][_0-9a-zA-Z\.%@]*)?)$/;
+const annRe = /^(@%|@%%|%@|[@:%]([_0-9a-zA-Z][_0-9a-zA-Z.%@]*)?)$/;
```

Comment:
just a note to validate that this is an intentional change (and not somehow related to formatting change)

Reply:
The backslash here is just redundant